packages/types/src/documents/app/datasource.ts

```diff
@@ -7,9 +7,7 @@ export interface Datasource extends Document {
   name?: string
   source: SourceName
   // the config is defined by the schema
-  config?: {
-    [key: string]: string | number | boolean | any[]
-  }
+  config?: Record<string, any>
```

Comment:
Why did we lose any?

Reply:
The typing of this was causing issues, it wasn't *really* accurate the type before, it was just a smattering of stuff, but in reality it could be absolutely anything, it depends on the datasource schema what is in the config, its basically impossible to type per datasource - each datasource implementation has an actual definition of the config.